### DISCUSSION TITLE
Implementing a Buy Now Pay Later service?

### DISCUSSION THREAD

**Discussion by soyzamudio** [swellstores/community#287] [Help] [answered]
I'm building a store (location: México), and I'm looking at integrating a Buy Now Pay Later service (similar to Klarna and Afterpay) called [Kueski Pay](https://www.kueskipay.com/)... is it possible to integrate it with Swell?

**Comment by sags95** [comment, marked as answer]
Hey @soyzamudio , yes Swell supports integrating with other payment methods and gateways besides those listed on our site. In this case it would be considered a custom payment method, we have a guide that's detailed [here](https://developers.swell.is/guides/integrate-payment-gateways-methods) which should prove useful. While now you would have to support some of the logic off-platform, in the future you'll be able to encapsulate the logic into a Swell App.

**Reply by soyzamudio** [reply]
Thanks a lot! I will take a look at the custom payment method!
